packages/vue/src/components/molecules/SfColorPicker/SfColorPicker.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="sf-color-picker" :class="{ 'sf-color-picker--active': isOpen }">
+    <transition name="sf-color-picker" mode="out-in">
+      <div v-if="!isOpen" key="1" class="sf-color-picker__open">
+        <slot name="open">
+          <SfButton
+            class="color-secondary sf-color-picker__open--button"
+            @click="toggle"
+            >+ Colors</SfButton
+          >
+        </slot>
+      </div>
+      <div v-else key="2" class="sf-color-picker__colors">
+        <slot name="label">
+          <div v-if="label" class="sf-color-picker__label">{{ label }}</div>
+        </slot>
+        <slot />
+        <slot name="close">
+          <SfIcon
```

Comment:
@martaradziszewska maybe we can use SfButton here?

Reply:
👌